src/importexport/musicxml/internal/musicxml/importmxmlnoteduration.h

```diff
@@ -61,6 +62,7 @@ class mxmlNoteDuration
     Fraction _dura;
     TDuration _normalType;
     Fraction _timeMod { 1, 1 };                       // default to no time modification
+    MusicXMLParserPass1* _pass1;
```

Comment:
better swap this with the next to avoid a compilerr warning reg. wrong order of initialisation